content/en/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -24,15 +24,7 @@
     <div class="row">
       <div class="col-md-9">
         <h2>Kubernetes Basics</h2>
-        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
-        <p>Using the interactive tutorials, you can learn to:</p>
-        <ul>
-        <li>Deploy a containerized application on a cluster.</li>
-        <li>Scale the deployment.</li>
-        <li>Update the containerized application with a new software version.</li>
-        <li>Debug the containerized application.</li>
-        </ul>
-        
+        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features, concepts and a tutorial for you to follow along.</p>
```

Comment:
```suggestion
        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features, concepts and a tutorial for you to follow along. These tutorials will let you manage a simple cluster and its containerized applications for yourself.</p>
        <p>Using these tutorials, you can learn to:</p>
        <ul>
        <li>Deploy a containerized application on a cluster.</li>
        <li>Scale the deployment.</li>
        <li>Update the containerized application with a new software version.</li>
        <li>Debug the containerized application.</li>
        </ul>
        
```

Reply:
Yes, that makes sense. Let me just change it.

Thanks